cvat-canvas/src/typescript/canvasModel.ts

```diff
@@ -288,15 +288,23 @@ export class CanvasModelImpl extends MasterImpl implements CanvasModel {
         const mutiplier = Math.sin((angle * Math.PI) / 180) + Math.cos((angle * Math.PI) / 180);
         if ((angle / 90) % 2) {
             // 90, 270, ..
-            this.data.top +=
-                mutiplier * ((x - this.data.imageSize.width / 2) * (oldScale / this.data.scale - 1)) * this.data.scale;
-            this.data.left -=
-                mutiplier * ((y - this.data.imageSize.height / 2) * (oldScale / this.data.scale - 1)) * this.data.scale;
+            // prettier-ignore
```

Comment:
Please, revert changes in files where aren't any significant changes, like this.

Reply:
As I mentioned this changes was made by prettier due to merge conflicts in merge. Also without this comments prettier and eslint was in conflict, so i added this comment.

Reply:
prettier & eslint are run only for git staged files. 
If you haven't changed them, they aren't linted.

Reply:
Please, try to checkout unchanged files to develop and commit them with --no-verify.
Example:
```
git checkout develop -- cvat-canvas/src/typescript/canvasModel.ts
git commit -m "test" --no-verify
```